kubernetes-addons/Observability/README.md

```diff
@@ -0,0 +1,151 @@
+# How-To Setup Observability for OPEA Workload in Kubernetes
+
+This guide provides a step-by-step approach to setting up observability for the OPEA workload in a Kubernetes environment. We will cover the setup of Prometheus and Grafana, as well as the collection of metrics for Gaudi hardware, OPEA/chatqna including TGI,TEI-Embedding,TEI-Reranking and other microservies, and PCM.
+
+#### Prepare
+
+```
+git clone https://github.com/opea-project/GenAIInfra.git
+cd kubernetes-addons/Observability
+```
+
+## 1. Setup Prometheus & Grafana
+
+Setting up Prometheus and Grafana is essential for monitoring and visualizing your workloads. Follow these steps to get started:
+
+### Step 1: Install Prometheus&Grafana
+
+```
+kubectl create ns monitoring
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm install prometheus-stack prometheus-community/kube-prometheus-stack --version 55.5.1 -n monitoring
```

Comment:
Btw.  Why version `55.5.1`?

Looking at output of `helm search repo -l kube-prometheus`...
* latest patchlevel in that series is `55.5.2`
  * https://github.com/prometheus-community/helm-charts/releases/tag/kube-prometheus-stack-55.5.2
* latest minor version in `55` series is `55.11.0`
  * With the same Prometheus-operator `v0.70.0` version (released year ago), as chart major version is increased when Prometheus-operator version is updated
* the very latest version is `62.6.0`, with Prometheus-operator `v0.76.1` (from last week)